### PR TITLE
Bi-grid coupled NA RRM configuration for E3SMv2

### PR DESCRIPTION
**Pull request by tangq**
Add the bi-grid (`--res northamericax4v1pg2_WC14to60E2r3_ICG`) NA RRM configuration for E3SMv2.

The test of this resolution and the A_WCYCL1850S_CMIP6 compset is successful (ongoing and completed 2+ years so far).
$rundir: /global/cscratch1/sd/tang30/E3SM_simulations/20210122.beta1_01.piControlSI.northamericax4v1pg2_WC14to60E2r3_ICG.cori-knl/run

A test (`SMS_PS.northamericax4v1pg2_WC14to60E2r3_ICG.A_WCYCL1850S_CMIP6.allactive-wcprodrrm`) is also added  to the production test suite for this grid.

[BFB]

### CODE REVIEW

**Review by rljacob — CHANGES_REQUESTED** [review]
Add an SMS test for this in the production test suite.

**Comment by tangq** [comment]
@singhbalwinder , as we talked at the RRMv2 meeting a couple of weeks ago, can you add the SMS test as requested by @rljacob ? I don't know how to add tests and can learn from this example. Thanks.

**Comment by singhbalwinder** [comment]
Sure. I will do that.

**Comment by singhbalwinder** [comment]
I created a test (`SMS.northamericax4v1pg2_WC14to60E2r3_ICG.A_WCYCL1850S_CMIP6`) but it is taking 77 nodes on Compy to run. With a PE layout of `S`, it takes about 25 nodes. If we do not have a coarser grid than the one we are using currently, I think this test should go into the production tests. If we have a coarser grid, we can see if we can fit it into our integration or developer suite.

**Comment by jonbob** [comment]
@singhbalwinder - I think the test is added for this specific grid, so it can't be more coarse

**Comment by rljacob** [comment]
Yes I said to put it in the production suite.  Not integration or developer.

**Comment by singhbalwinder** [comment]
@rljacob : You did mention production suite, sorry I missed that.

@tangq : I have added the test. Some important points to note:

1. I removed the history output variables from all user_nl_* files as I was getting into errors. Some variables mentioned in the `fincl` lists were missing in the model. We can revisit this and add the ones which work if you like. I have removed all history output for now (from atm and land user_nl_* files)
2. It seems like some input files are present on Compy but not in our SVN input data server. I tried running on Anvil and the model crashed as model was not able to find some files on the SVN server. We need to fix that before merging this PR
3. I choose "small" PE layout (`SMS_PS`) as my test runs sometime waited for almost a day in queue if I use the default PE layout. 
4. This test runs for 5 days (by default), if you would like to reduce the runtime, we can change that easily.

**Comment by tangq** [comment]
@singhbalwinder , thanks for adding the test. Regarding the notes,

1. My simulation with the history output is still running on cori (/global/cscratch1/sd/tang30/E3SM_simulations/20210122.beta1_01.piControlSI.northamericax4v1pg2_WC14to60E2r3_ICG.cori-knl). It is not clear to me why the history output doesn't work for the test. 
@rljacob , do you think we should keep the history output in the test or not?
2. What files are the missing on the input data server? If you can send me the file list, I can add them to the server.
3. On cori-knl, I often need to wait 1 day or so before the 72-node job starts.
4. If we reduce the simulation length, does it fit in the debug queue? If so, I think we should shorten the test.

File: cime_config/tests.py [hunk 1]
```diff
@@ -211,7 +211,10 @@
     #e3sm tests to mimic production runs
     "e3sm_prod" : {
         "inherit" : "e3sm_atm_prod",
-        "tests"   : "SMS_Ld1.ne30pg2_r05_EC30to60E2r2-1900_ICG.A_WCYCL1850S_CMIP6.allactive-wcprod"
+        "tests"   : (
+            "SMS_Ld1.ne30pg2_r05_EC30to60E2r2-1900_ICG.A_WCYCL1850S_CMIP6.allactive-wcprod",
+            "SMS_PS.northamericax4v1pg2_WC14to60E2r3_ICG.A_WCYCL1850S_CMIP6.allactive-v2_wcprod",
+            )
```
Comment (rljacob) [inline cime_config/tests.py:217]: Please don't put "v2" in the testmod.  change it to "wcprodrrm"

**Comment by rljacob** [comment]
No you don't have to include those history fields.  The testmod should reflect the production settings but first it has to work.

**Review by rljacob — APPROVED** [review]

**Comment by singhbalwinder** [comment]
@tangq : I have now added the history fields back for all components. I missed turning on COSP, that is why I was getting those errors about the `fincl` fields. I have now turned on COSP and everything works as expected. I also moved the missing files to the file server and added a PE layout for Anvil. The test now works on both Compy and Anvil.

**Comment by tangq** [comment]
@singhbalwinder , ah, that makes sense now. The COSP output is included in the production run.

Can this PR be merged to master now? Thanks!

**Comment by singhbalwinder** [comment]
Production tests are only run on Compy and Anvil. In that sense, this PR won't break any tests. So , I think, this should be ready to merge.

**Comment by tangq** [comment]
@singhbalwinder , we talked about this PR at the RRMv2 telecon yesterday and planned to do further tests after this PR is merged. I see @rljacob already approved this PR, so it should be ready to be merged. Can you merge it when you have time? Thanks.

**Comment by singhbalwinder** [comment]
Sure. I think this PR is BFB but let me know if it isn't. I will start merging it today.

**Comment by singhbalwinder** [comment]
Merged to next

**Pull request merged**